lib/flux/app-state.ts

```diff
@@ -74,7 +73,7 @@ export const actionMap = new ActionMap({
 
       return update(state, {
         showNavigation: { $set: true },
-        showNoteInfo: { $set: false },
+        showNoteInfo: { $set: false }, // @todo?
```

Comment:
I think this is safe to remove (how do you activate the navigation panel without also triggering the note info "click outside" action?) but if not this needs to figure out how to set state.ui.showNoteInfo